Documentation for the Remootio integration

ivgg-me opened the pull request:
## Proposed change

Documentation for the [Remootio](https://www.remootio.com/) integration introduced with the PR home-assistant/core#63155.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core#63155
- Link to parent pull request in the Brands repository: home-assistant/brands#3035
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

ivgg-me committed:
Documentation for the Remootio integration

homeassistant commented:
Hi @ivgg-me, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20970).

Once you do that we will be able to review and accept this pull request.

Thanks!

klaasnicolaas commented on source/_integrations/remootio.markdown:
```diff
@@ -0,0 +1,23 @@
+---
+title: Remootio
+ha_release: ???
+ha_category:
+  - Cover
+ha_iot_class: Local Push
+ha_domain: remootio
+ha_codeowners:
+  - '@ivgg-me'
+ha_config_flow: true
```
missing `description` and `ha_platforms`

ivgg-me replied:
I have created my documentation according the [documentation of Home Assistant](https://developers.home-assistant.io/docs/documenting/create-page). Is this part of the Home Assistant documentation out of date?

ivgg-me replied:
Fixed.

klaasnicolaas replied:
it's a little bit out of date 😉

frenck commented on source/_integrations/remootio.markdown:
```diff
@@ -0,0 +1,23 @@
+
+### Requirements
+
+To use this integration with your Remootio device it must have at least the software version _2.21_, has to be connected to your Wi-Fi, must have a fixed IP address assigned and 
+a status sensor installed. Furthermore the API access must be enabled on the device.
+
+To enable the API access on your Remootio device you need to access it using the master key via the Remootio app, afterwards go to the _Websocket API_ settings 
+in the _Device software_ section of the device settings and enbale the API with logging.
```
```suggestion
To use this integration with your Remootio device it must have at least the software version _2.21_, has to be connected to your Wi-Fi, must have a fixed IP address assigned and 
a status sensor installed. Furthermore, the API access must be enabled on the device.

To enable the API access on your Remootio device you need to access it using the master key via the Remootio app, afterward go to the _Websocket API_ settings 
in the _Device software_ section of the device settings and enable the API with logging.
```

ivgg-me committed:
Update source/_integrations/remootio.markdown

Based on the suggestions of @frenck; see https://github.com/home-assistant/home-assistant.io/pull/20970#pullrequestreview-842426960

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

ivgg-me committed:
Changes requested by @klaasnicolaas

Changes requested by @klaasnicolaas; see https://github.com/home-assistant/home-assistant.io/pull/20970#pullrequestreview-842354101

ivgg-me committed:
Documentation of events

ivgg-me committed:
Documentation of events

frenck commented on source/_integrations/remootio.markdown:
```diff
@@ -24,3 +24,29 @@ To enable the API access on your Remootio device you need to access it using the
+
+## Events
+
+The integration fires the event `remootio_left_open` if the Remootio controlled garage door or garden gate has been left open for 5 minutes.
```
Can we use device actions instead?

ivgg-me replied:
According to the [documentation](https://developers.home-assistant.io/docs/device_automation_action/) device actions allow a user to have a device do something. 

In this case the Remootio device fires an event if the controlled garage door or garden gate has been left open for 5 minutes. Based on this event the integration fires the `remootio_left_open` [Home Assistant event](https://developers.home-assistant.io/docs/integration_events).

frenck replied:
> In this case the Remootio device fires an event if the controlled garage door or garden gate has been left open for 5 minutes. 

Device actions are using events under the hood, but provide UI controls to interact with those.

<https://developers.home-assistant.io/docs/device_automation_index>

ivgg-me replied:
If you mean that this is really needed for the first version of the integration, please make then an request for change in home-assistant/core#63155. Thank you!

frenck replied:
Maybe to much for a first version indeed. however, lets at least not use json here.

ivgg-me replied:
> Maybe to much for a first version indeed. however, lets at least not use json here.

Sorry, I don't understand. What you mean with _not use json here_?

frenck replied:
Nevermind it is indeed the raw event output.  👍

frenck commented:
Can you please make sure the build passes?

ivgg-me commented:
> Can you please make sure the build passes?

I don't understand the errors that occurred during the build. Is there a documentation?

Commit:
Fix for build problems

Commit:
Fix for failed checks

ivgg-me commented:
> Can you please make sure the build passes?

The checks are now running successfully.

ivgg-me commented:
@frenck What else is necessary for this integration to be merged?

frenck commented:
@ivgg-me This is the documentation PR, it won't be merged if the parent PR in the core code base isn't merged yet. This PR is thus dependent on and awaiting that.

ivgg-me committed:
Documentation for Remootio integration
